Neos.FluidAdaptor/Tests/Functional/Core/Fixtures/ViewHelpers/RedirectViewHelper.php

```diff
@@ -33,6 +34,7 @@ class RedirectViewHelper extends AbstractWidgetViewHelper
      */
     public function render()
     {
-        return $this->initiateSubRequest();
+        $response = $this->initiateSubRequest();
```

Comment:
Ouch, we actually have a test that verifies that a widget can cause a redirect... this ofc changes things regarding my feedback on the AbstractWidgetViewHelper in case this is really something we DO WANT.

Reply:
I still don't like it, probably something to discuss at the sprint, I guess I will leave the stuff as it is now and take care of the other open points more close to the intention of the PR, and we can revisit this. I would prefer to minimise the API for sub requests and really think through what we want to do with them. It just comes to my mind that backend modules are also using sub requests in a quite weird way, might be that they actually need way more than content. Anyway, this kind of works for now and at least makes the transition from main action dispatch to HTTP relatively clear.